backend/api/views/main.py

```diff
@@ -80,7 +85,12 @@ def get_document():
     else:
         docs = Document.query.filter_by(**kwargs)
         if description is not None:
-            docs = [i for i in docs if description.lower() in i.description.lower()]
+            docs = [
+                i
+                for i in docs
+                if i.description is not None
+                and description.lower() in i.description.lower()
+            ]
         if date is not None:
             docs = [i for i in docs if date.lower() in str(i.date).lower()]
         if len(serialize_list(docs)) == 0:
```

Comment:
Woud like some more comments here - it's a bit hard to tell what exactly is going on here vs. the code afterwards for exact match.